CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ Added
 - Added a UI button for redeploying an EVC.
 - UNI tag_type are now accepted as string.
 - EVCs now listen to ``switch.interface.(link_up|link_down|created|deleted)`` events for activation/deactivation
+- Circuits with a vlan range are supported now. The ranges follows ``list[list[int]]`` format and both UNIs vlan should have the same ranges.
```

Comment:
If you could also mention in the changelog that the openapi has been updated accordingly and for more information users should check it out, that'd be great.

Reply:
Updated changelog, commit [918e21324fdd2dc3d6615cf967613ae75802fb31](https://github.com/kytos-ng/mef_eline/pull/407/commits/918e21324fdd2dc3d6615cf967613ae75802fb31)